tests/components/canary/test_sensor.py

```diff
@@ -82,6 +84,12 @@ async def test_sensors_pro(hass, canary) -> None:
         assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == data[2]
         assert state.state == data[1]
 
+    device = device_registry.async_get_device({(DOMAIN, "20")}, set())
+    assert device
+    assert device.manufacturer == MANUFACTURER
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6184085)